### ISSUE TITLE
aria-hidden nested inside a `<label>` is being ignored by JAWS

### ISSUE THREAD

**Issue by corinnaSchultz** [FreedomScientific/VFO-standards-support#109]
### Summary

I have a `<label>` which contains a `<span aria-hidden="true">` child element. The contents of the span are being read by JAWS.

See markup below.

### Expected result

The contents of the span should be ignored because of aria-hidden.

### Actual result

The contents of the span are being read.

### Example

```
<label for="session-standard" class="ng-binding">Session Name 
  <span class="es ng-binding" aria-hidden="true"> Nombre de sesion</span>
</label>
```

#### JAWS version and build number
JAWS 2018.1807.8 ILM

#### Operating System and version
Windows 10

#### Browser and version:
Firefox 61 (not sure of the minor version)

**Comment by stevefaulkner** [comment]
test case https://s.codepen.io/stevef/debug/VGzJRO

**Comment by stevefaulkner** [comment]
When testing using the test case in https://github.com/FreedomScientific/VFO-standards-support/issues/109#issuecomment-418990212 `aria-hidden` works as expected when the label recieves virtual focus in JAWS only the non aria-hidden text is announced "Session Name ". It also works as expected when the `text input` recieves focus. i.e. for the purposes of calculating an accessible name for the control, `aria-hidden` is ignored. So "Session Name Nombre de sesion" is announced

refer to:  [Hiding Content Has No Effect on Accessible Name or Description Calculation](https://w3c.github.io/using-aria/#hiding-content-has-no-effect-on-accessible-name-or-description-calculation)

**Comment by stevefaulkner** [comment]
2 possible methods to acheive the desired effect
Move the aria-hidden content out of the `label`
`<label for="session-standard" class="ng-binding">Session Name</label> <span class="es ng-binding" aria-hidden="true"> Nombre de sesion</span>`

add an `aria-label` to the `text input`
```<label for="session-standard" class="ng-binding">Session Name 
  <span class="es ng-binding" aria-hidden="true"> Nombre de sesion</span>
</label>

<input type="text" id="session-standard" aria-label="Session Name">
```

**Comment by corinnaSchultz** [comment]
Thank you, this is helpful.

**Comment by stevefaulkner** [comment]
@corinnaSchultz no worries
